addon/components/collapsable-facet/template.hbs

```diff
@@ -9,25 +9,30 @@
   </div>
   <div class="{{options.groupListWrapperClass}} collapse{{if open " in"}}" id="{{facetId}}">
     <div class="{{options.groupListClass}}">
-    {{#each facet.terms as |term|}}
-      <div class="term">
-        <span class="term">
-        {{#if facet.link}}
-          <a href="#" class="{{if term.selected "active"}}" {{action "toggleFacet" term}}>{{if term.display term.display term.value}}</a>
-        {{else if facet.checkbox}}
-          <label>
-          {{input type="checkbox" checked=term.selected click=(action "toggleFacet" term)}}
-            {{if term.display term.display term.value}}
-            </label>
+      {{#each facet.terms as |term|}}
+        <div class="term">
+          <span class="term">
+          {{#if facet.link}}
+            <a href="#" class="{{if term.selected "active"}}" {{action "toggleFacet" term}}>{{if term.display term.display term.value}}</a>
+          {{else if facet.checkbox}}
+            <label>
+            {{input type="checkbox" checked=term.selected click=(action "toggleFacet" term)}}
+              {{if term.display term.display term.value}}
+              </label>
+          {{else if facet.datetime}}
+            <div class="form-group form-inline">
+              <label for="filter-from-time">{{term.display}} </label>
```

Comment:
The `<label>`'s `for` will match two things here, in your case : we should make this unique for each `term` in `Facet.type = datetime`  ; maybe employ `{{#each facet.terms as |term index|}}` and use `<label for="datetime-{{index}}">`.  Though, does `bs-datetimepicker` render an input w/ a specific ID that matches your `for`?